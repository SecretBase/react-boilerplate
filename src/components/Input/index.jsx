// @flow
import styled from 'styled-components'

const Input = styled.input`
  border: 1px solid #ddd;
  max-width: 980px;
  width: 100%;
  margin: 16px auto;
  box-sizing: border-box;
  display: block;
  font-size: 24px;
  padding: 8px 16px;
`

export default Input
