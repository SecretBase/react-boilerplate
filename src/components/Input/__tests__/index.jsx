import React from 'react'
import {shallow} from 'enzyme'

import Input from '../index'

describe('Input component', () => {
  it('should render correctlt', () => {
    const wrapper = shallow(
      <Input />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
