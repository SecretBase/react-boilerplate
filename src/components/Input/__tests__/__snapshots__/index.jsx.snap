// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should render correctlt 1`] = `
<input
  className="sc-bdVaJa glQsoy"
/>
`;
